Libraries/ReactIOS/verifyPropTypes.js

```diff
@@ -24,7 +24,8 @@ function verifyPropTypes(
   }
   var nativeProps = viewConfig.nativeProps;
   for (var prop in nativeProps) {
-    if (!component.propTypes[prop] &&
+    if (component.propTypes != undefined && 
```

Comment:
Please use `===` or `!==` instead of their looser counterparts whenever possible.